### ISSUE TITLE
[DeepLab v3+] Hardware configuration and hyper-parameters to reproduce results on Cityscapes

### ISSUE THREAD

**Issue by mtli** [tensorflow/models#4821]
Hi, I am trying to reproduce the 78.79% of the released xception_cityscapes_trainfine and it seems that the hyper-paramters listed in https://github.com/tensorflow/models/blob/master/research/deeplab/g3doc/cityscapes.md is for demonstrative purpose.

What are the hardware configuration (e.g. how many GPUs and vRAM per GPU) and the hyper-parameters (e.g. batch size, number of steps, learning rate) to reproduce such result on Cityscapes?

Have I written custom code: N/A
OS Platform and Distribution: N/A
TensorFlow installed from: N/A
TensorFlow version: N/A
Bazel version: N/A
CUDA/cuDNN version: N/A
GPU model and memory: N/A
Exact command to reproduce: N/A

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by baoruxiao** [comment]
Hi @mtli 

Have you successfully reproduce the performance? Mine only get 74.7% with 0.007 base learning rate.

**Comment by mtli** [comment]
Hi @baoruxiao, I am not able to reproduce the result either.

**Comment by drorsimon** [comment]
I got to 77% using batch size 12 and learning rate of 1e-2.
Unfortunately, this is still really far away from their published 78.7%.
Would also appreciate more information about the exact software/hardware setup to be able to reproduce these results.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
